src/Model/Behavior/DuplicatableBehavior.php

```diff
@@ -176,7 +178,7 @@ protected function _getContain()
     protected function _modifyEntity(EntityInterface $entity, $object)
     {
         // belongs to many is tricky
-        if ($object instanceof BelongsToMany) {
+        if ($object instanceof BelongsToMany && ! $this->getConfig('preserveJoinData')) {
```

Comment:
Any reason for the space after the `!`

Reply:
Easier to read in my opinion. I tend to lose "!", while in "!$" context. I was also convinced that it's part of PSR Coding Style Guide, but can't find it.